packages/shared/forks/ReactFeatureFlags.testing.www.js

```diff
@@ -77,6 +77,7 @@ export const enableFloat = false;
 export const enableHostSingletons = false;
 
 export const useModernStrictMode = false;
+export const enableFizzExternalRuntime = false;
```

Comment:
Would this be `true` instead?

Reply:
I think this is dead code tbh. I don't think we use it anywhere in www. It's a vestige from when we momentarily thought we would provide a special testing build of React. Need to delete, just haven't yet.